app/src/main/res/values/strings.xml

```diff
@@ -85,7 +86,7 @@
     <string name="iris_scan_right_title">Iris scan - Right eye</string>
     <string name="participant_matching_title">Child matching</string>
     <string name="participant_flow_child_number_title">Child Number</string>
-    <string name="participant_flow_participant_id_title">Child ID</string>
+    <string name="participant_flow_participant_id_title">Child ID*</string>
```

Comment:
@sherrif10 this property is used in whole app. How often do I have to repeat this?..